app/src/main/res/menu/menu_user_detail.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <!-- Search, should appear as action button -->
+    <item
+        android:id="@+id/action_edit"
+        android:animateLayoutChanges="true"
+        android:icon="@drawable/ic_edit_white"
+        android:title="@string/delete"
+        app:showAsAction="collapseActionView|always"/>
```

Comment:
WARNING: [AlwaysShowAction] Prefer "`ifRoom`" instead of "`always`"
 Using `showAsAction="always"` in menu XML, or `MenuItem.SHOW_AS_ACTION_ALWAYS` in Java code is usually a deviation from the user interface style guide.Use `ifRoom` or the corresponding `MenuItem.SHOW_AS_ACTION_IF_ROOM` instead.

If `always` is used sparingly there are usually no problems and behavior is roughly equivalent to `ifRoom` but with preference over other `ifRoom` items. Using it more than twice in the same menu is a bad idea.

This check looks for menu XML files that contain more than two `always` actions, or some `always` actions and no `ifRoom` actions. In Java code, it looks for projects that contain references to `MenuItem.SHOW_AS_ACTION_ALWAYS` and no references to `MenuItem.SHOW_AS_ACTION_IF_ROOM`.